src/Symfony/Component/Translation/TranslatorBagInterface.php

```diff
@@ -21,11 +21,11 @@
     /**
      * Gets the catalogue by locale.
      *
-     * @param string|null $locale The locale or null to use the default
+     * @param string $locale The locale
      *
      * @throws \InvalidArgumentException If the locale contains invalid characters
      *
      * @return MessageCatalogueInterface
      */
-    public function getCatalogue($locale = null);
+    public function getCatalogue($locale);
```

Comment:
I'm not sure if this can break the old implementation, we can still use the old signature without any issue like `Translator` class.